Source/WebCore/dom/FullscreenOptions.idl

```diff
@@ -31,8 +31,17 @@
     "hide"
 };
 
+[
+    Conditional=FULLSCREEN_API
```

Comment:
We might need a different conditional, just to keep things separated... but it's a start.